src/helpers/Activity_Wheel.java

```diff
@@ -1,60 +1,115 @@
+package helpers;
+
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.Random;
-import javax.swing.JOptionPane;
+import java.util.Scanner;
+
 public class Activity_Wheel {
-    public static void pickActivity() {
-        //variable "dice" acts as the randomizer depending on the int (number) given (shown later on)
+
+    private static String activityMessage; // may not need this anymore
+
+    /**
+     * Returns a String message containing an activity that is randomly selected from an ArrayList
+     * of possible activities, which is filled based on the weather String passed to the method
+     *
+     * @param weather a String containing a type of weather condition obtained from the weatherAPI
+     * @return a String message
+     * @throws FileNotFoundException
+     */
+    public static String generateActivity(String weather) throws FileNotFoundException {
```

Comment:
Depending on the weather condition passed to the method, it will call the generateList method to fill an ArrayList using the text from a file corresponding to that weather condition